pyscript.core/src/plugins/deprecations-manager.js

```diff
@@ -0,0 +1,22 @@
+// PyScript Derepcations Plugin
+import { hooks } from "../core.js";
+import { notify } from "./error.js";
+
+// eslint-disable-next-line no-unused-vars
+hooks.main.onReady.add(function checkDeprecations(pyScript) {
+    const scripts = document.querySelectorAll("script");
+    for (const script of scripts) checkLoadingScriptsFromLatest(script.src);
+});
+// eslint-disable-next-line no-unused-vars
+
+/**
+ * Check if src being loaded from pyscript.net/latest and display a notification if true
+ * * @param {string} src
+ */
+function checkLoadingScriptsFromLatest(src) {
+    if (/\/pyscript\.net\/latest/.test(src)) {
+        notify(
+            "Loading scripts from latest is deprecated and will be removed soon. Please use a specific version instead.",
```

Comment:
Maybe we add a link to the GitHub readme or similar here, so it's easy for users to find the current release?

Reply:
Good call but I'm not sure where though. Maybe to the docs?

In general I always thought we missed a "releases" page with some additional metadata

Reply:
Yeah there was a user (on here? Discord?) who requested such a thing... it seems like a good idea. 

Link to the docs is a good idea... or maybe the [GitHub releases](https://github.com/pyscript/pyscript) page, though that doesn't just have a pre-formatted link... maybe the [First Steps](https://docs.pyscript.net/latest/user-guide/first-steps/) page in the docs?

Reply:
Sounds like a plan. [GH Releases ](https://github.com/pyscript/pyscript/releases) is something but honestly I'm not a big fan but better nothing :)